opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -428,9 +435,9 @@ namespace Opm
 
         const Wells* wells = wellsmanager.c_wells();
         wellstate.resize(wells, simulatorstate); //Resize for restart step
-        auto restarted = Opm::init_from_restart_file(
-                                eclipseState_,
-                                Opm::UgGridHelpers::numCells(grid) );
+        auto restarted = Opm::load_from_restart_file(eclipseState_,
```

Comment:
I don't see a freestanding `load_from_restart_file()` function in OPM/opm-output#152.  Did you mean the method
```C++
Opm::EclipseIO::load_from_restart_file()
```
instead?